libkbfs/keybase_service_base.go

```diff
@@ -619,3 +619,25 @@ func (k *KeybaseServiceBase) GetTLFCryptKeys(ctx context.Context,
 
 	return res, nil
 }
+
+// GetTLFCanonicalID implements the TlfKeysInterface interface for
+// KeybaseServiceBase.
+func (k *KeybaseServiceBase) GetTLFCanonicalID(ctx context.Context,
```

Comment:
What does "canonical ID" mean?  I don't think we have different ways to present a TLF ID.  I guess you're referring to the name?  Should this be `GetCanonicalTLFNameAndID`?